lib/browser/highlightTags.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
please rename to `highlight-tags.js` or `highlight_tags.js`

case-insensitive filesystems suck